### DISCUSSION TITLE
YouTube Channel Logo Is Not Needed I Guess

### DISCUSSION THREAD

**Discussion by 0xSaurabhx** [libre-tube/LibreTube#1200] [Ideas]
![Screenshot_2022-08-29-21-53-48-83.jpg](https://user-images.githubusercontent.com/61317144/187248672-9d10cd55-4130-46df-9f99-4417d28b5125.jpg)

Here, I guess Placing YouTube Channel Logo With It's Name Makes Lack Of Spaces In Between So, LibreTube Team Please Remove Them Make It Look Clean.

And Also Make The YouTube Video Title Size Should be Bigger & Bolder Than Channel Name, This Is Make It Better I Hope So.

**Comment by FireMasterK** [comment]
I disagree, the YouTube app does this too, and I think it looks fine.

The only place where it shouldn't be visible is channels imo.

**Reply by 0xSaurabhx** [reply]
LOL In Channel Section It Should Be Visible. Ofc It's It's Main Section

**Reply by Bnyro** [reply]
Maybe we should lower the size of the avatar but I don't think that removing it is a good idea.

**Reply by 0xSaurabhx** [reply]
Yeah I think This Is Good IDEA!
